sos/report/plugins/ipmitool.py

```diff
@@ -27,12 +27,17 @@ def setup(self):
             cmd += " -I usb"
 
         self.add_cmd_output([
+            "%s channel info 3" % cmd,
+            "%s channel getaccess 3" % cmd,
+            "%s raw 0x30 0xb0" % cmd,
+            "%s raw 0x30 0x65" % cmd,
```

Comment:
I'm not familiar with what `0x30`, `0xb0`, or `0x65` map to. I know they're ipmi commands, but I'm just not sure what commands they are.

Can you please detail what these are? We'd likely be well served sticking that in a comment as well.

Reply:
From Intel's guide:
`0x30 0x65 = Get HDD drive Fault LED State`
This command retrieves the drive LED states for each slot. The second returned data is in the same format as the first request data in the Set HDD drive fault LED command.

`0x30 0xb0 = Get LED Status`
Retrieves the LED status.
Status definitions:00b = Off01b = Blink10b = On11b = Invalid (not installed)